infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/bootstrapper/OpenShiftBootstrapper.java

```diff
@@ -103,7 +103,8 @@ protected void doBootstrapAsync(String installerWebsocketEndpoint, String output
             + Integer.toString(installerTimeoutSeconds)
             + " -file "
             + BOOTSTRAPPER_DIR
-            + CONFIG_FILE);
+            + CONFIG_FILE
+            + " &>/dev/null &");
```

Comment:
Please add a comment why you've added this. It is not obvious that it is a way to make process detached.